jekyll/_cci2/configuration-cookbook.md

```diff
@@ -65,89 +60,68 @@ Configuration Recipe | Description
+**Note:** the `CIRCLE_SHA1` variable used in this example is built-in, so is always available.
 
-#### Updating the Amazon ECS Sevice Without Updating AWS CLI
+### Build, Push and Deploy a Service Update
 
-To update the Amazon ECS Service without updating the AWS CLI using CircleCI orbs, review the the example shown below, which shows you how to update the ECS service.
+To configure an [AWS service update](https://docs.aws.amazon.com/AmazonECS/latest/developerguide/update-service.html) to deploy a newly built image from AWS ECR, you can use orbs to keep your configuration as simple as possible: the `aws-ecr` orb to build and push an updated image to ECR, and the `aws-ecs` orb to deploy you service update.
+
+The following example shows building and pushing an image to AWS ECR and pushing that image as a service update to AWS ECS:
+ 
+```yml
+version: 2.1 # 2.1 config required to use orbs
 
-```yaml
-version: 2.1
 orbs:
-  aws-ecr: circleci/aws-ecr@0.0.4
-  aws-ecs: circleci/aws-ecs@0.0.3
+  aws-ecr: circleci/aws-ecr@x.y.z # invoke the AWS ECR orb
+  aws-ecs: circleci/aws-ecs@x.y.z # invoke the AWS ECS orb
+
 workflows:
   build-and-deploy:
     jobs:
-      - aws-ecr/build-and-push-image:
-          account-url: '${AWS_ACCOUNT_ID}.dkr.ecr.${AWS_REGION}.amazonaws.com'
+      - aws-ecr/build-and-push-image: # orb built-in job
           repo: '${MY_APP_PREFIX}'
-          region: '${AWS_REGION}'
           tag: '${CIRCLE_SHA1}'
-      - aws-ecs/deploy-service-update:
+      - aws-ecs/deploy-service-update: # orb built-in job
           requires:
             - aws-ecr/build-and-push-image
           family: '${MY_APP_PREFIX}-service'
           cluster-name: '${MY_APP_PREFIX}-cluster'
           container-image-name-updates: 'container=${MY_APP_PREFIX}-service,tag=${CIRCLE_SHA1}'
 ```
 
-Notice in this example that you need to instantiate two different AWS ECS orbs: `aws-ecs 0.0.3` and `aws-ecr 0.0.4` to update the ECS service. Once you have instantiated these two orbs, the orb enables the configuration, and then pushes the image, before finally deploying the service update to ECS.
+For a full list of usage options and orb elements see the [AWS-ECS orb page](https://circleci.com/orbs/registry/orb/circleci/aws-ecs) in the CircleCI Orbs Registry.
 
-#### Updating the Amazon Web Services CLI and Amazon ECS
+### Verifying the Amazon ECS Service Update
 
-To update both the AWS CLI and ECS Service simultaneously, use the orb shown below to simplify the process of updating these services.
+Once you have updated the Amazon ECS service, verify the update was properly applied using the CircleCI AWS CLI and ECS orbs. This orb example is shown below, this time, rather than using a built-in job to perform the required process, commands from the orbs are used as steps in the job definition for a job called `verify-deployment`. 
```

Comment:
```suggestion
Once you have updated the Amazon ECS service, you can verify the update was correctly applied. To keep your config as simple as possible, use the AWS CLI and ECS orbs. This time, rather than using an orb's built-in job to perform the required process, commands from the orbs are used as steps in the definition of the job named `verify-deployment`. 
```